src/Tizen.NUI/src/internal/WebView.cs

```diff
@@ -254,6 +263,17 @@ public WebView(string locale, string timezoneId) : this(Interop.WebView.New2(loc
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
         }
 
+        /// <summary>
+        /// Creates an uninitialized WebView.
+        /// <param name="argc">count of args</param>
+        /// <param name="args">args array</param>
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public WebView(int argc, string[] args) : this(Interop.WebView.New3(argc, args), true)
```

Comment:
I don't think the first argument `argc` is necessary. It can be optained from args.Length.